sim/src/main/scala/midasexamples/PlusArgsModule.scala

```diff
@@ -0,0 +1,59 @@
+//See LICENSE for license details.
+
+package firesim.midasexamples
+
+import chisel3._
+import chisel3.util._
+import chisel3.util.Enum
+import freechips.rocketchip.config.{Config, Field, Parameters}
+
+import midas.widgets._
+
+import midas.widgets.ResetPulseBridge
+
+/** Defines a test group with the id of 0
+  */
+class PlusArgsModuleTestConfigGroup68Bit
+    extends Config((site, here, up) => { case PlusArgsTestNumber =>
+      0
+    })
+
+/** Defines a test group with the id of 1
+  */
+class PlusArgsModuleTestConfigGroup29Bit
+    extends Config((site, here, up) => { case PlusArgsTestNumber =>
+      1
+    })
+
+case object PlusArgsTestNumber extends Field[Int]
```

Comment:
```suggestion
case object PlusArgsTestNumberKey extends Field[Int]
```
I'm trying to follow the federation/RC convention of suffixing `Field`s with `Key`. Not something you could've known a priori.